app/src/main/java/com/nextcloud/talk/conversationlist/ConversationsListActivity.kt

```diff
@@ -446,6 +449,34 @@ class ConversationsListActivity :
         updateFilterConversationButtonColor()
     }
 
+    private fun List<ConversationModel>.refreshMessages() {
```

Comment:
As we already have a ConversationListViewModel we should use this, so as the code is just about data handling it should not be in the Activity.